tests/Unit/PrestaShopBundle/Service/Multistore/CustomizedConfigurationCheckerTest.php

```diff
@@ -0,0 +1,76 @@
+<?php
+/**
+ * Copyright since 2007 PrestaShop SA and Contributors
+ * PrestaShop is an International Registered Trademark & Property of PrestaShop SA
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.md.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to https://devdocs.prestashop.com/ for more information.
+ *
+ * @author    PrestaShop SA and Contributors <contact@prestashop.com>
+ * @copyright Since 2007 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ */
+
+namespace Tests\Unit\PrestaShopBundle\Service\Multistore;
+
+use PHPUnit\Framework\MockObject\MockObject;
+use PHPUnit\Framework\TestCase;
+use PrestaShop\PrestaShop\Core\Domain\Configuration\ShopConfigurationInterface;
+use PrestaShopBundle\Entity\Shop;
+use PrestaShopBundle\Entity\ShopGroup;
+use PrestaShopBundle\Service\Multistore\CustomizedConfigurationChecker;
+use Prophecy\Prophecy\ObjectProphecy;
+
+class CustomizedConfigurationCheckerTest extends TestCase
+{
+    public function testIsConfigurationCustomizedForThisShop(): void
+    {
+        $customizedConfigurationChecker = new CustomizedConfigurationChecker($this->mockShopConfiguration(true));
+        $res = $customizedConfigurationChecker->isConfigurationCustomizedForThisShop('FAKE_CONFIG_KEY', $this->prophesizeShopEntity()->reveal());
+        $this->assertTrue($res);
+
+        $customizedConfigurationChecker = new CustomizedConfigurationChecker($this->mockShopConfiguration(false));
+        $res = $customizedConfigurationChecker->isConfigurationCustomizedForThisShop('FAKE_CONFIG_KEY', $this->prophesizeShopEntity()->reveal());
+        $this->assertFalse($res);
```

Comment:
```suggestion
        $this->assertFalse($customizedConfigurationChecker->isConfigurationCustomizedForThisShop('FAKE_CONFIG_KEY', $this->prophesizeShopEntity()->reveal()));
```

Reply:
Too long lines are harder to read in an IDE or GitHub, which is why [PSR recommends](https://www.php-fig.org/psr/psr-2/) between 120 characters and 80 characters or less.

I think this is an unneeded optimization.